rcsb/db/processors/DataExchangeStatus.py

```diff
@@ -15,8 +15,9 @@
 
 
 import logging
-
-from rcsb.db.utils.TimeUtil import TimeUtil
+from datetime import datetime
+from typing import Optional
+from zoneinfo import ZoneInfo
```

Comment:
Likewise, if we keep `TimeUtil`, we don't have to worry about updating each of these files that currently imports it.